invisible_cities/cities/base_cities.py

```diff
@@ -159,16 +171,25 @@ def drive(cls, argv):
         if opts.print_config_only:
             return
         instance = cls(**conf.as_dict)
+
+        # set the deamons
+
+        if 'daemons' in conf.as_dict:
+            d_list_name = conf.as_dict['daemons']
```

Comment:
Whatever was being smoked while writing this, I want some!

Fixes coming up, when I get a bit more spare time ...